### PR TITLE
Comments & Formatting improvements

### PR DESCRIPTION
**Pull request by eddiejaoude**
- [x] Added missing docblocs
- [x] Fixed incorrect docblocs
- [x] Fixed formatting
- [x] Added TODO for DI

### CODE REVIEW

File: src/Elfiggo/Brobdingnagian/Param/Params.php [hunk 1]
```diff
@@ -25,18 +29,28 @@ class Params
      */
     private $input;
 
-
+    /**
+     * @param ServiceContainer $serviceContainer
+     *
+     * @TODO: should inject the ServiceContainer, inject required only
```
Comment (Elfiggo) [inline src/Elfiggo/Brobdingnagian/Param/Params.php]: You could implement this ;)

Comment (eddiejaoude) [inline src/Elfiggo/Brobdingnagian/Param/Params.php]: This will be done in a later PR ;)

**Comment by eddiejaoude** [comment]
@Elfiggo I removed the TODO as requested and created an issue

**Comment by Elfiggo** [comment]
Thanks.

**Pull request merged**